File: Batch-Compliance/js/download_compliance_results.js
const needle = require('needle');

// Replace with your job download_url
downloadUrl = ''

async function getRequest() {

    const res = await needle('get', downloadUrl, { compressed: true })

    if (res.body) {
        return res.body.toString('utf8');
    } else {
        throw new Error('Unsuccessful request');
    }
}

(async () => {

    try {
        // Make request
        const response = await getRequest();
        console.dir(response, {
            depth: null
        });

    } catch (e) {
        console.log(e);
        process.exit(-1);
    }
    process.exit();
})();

File: Batch-Compliance/python/download_compliance_results.py
import requests

# Replace with your job download_url
download_url = ''


def connect_to_endpoint(url):
    response = requests.request("GET", url)
    print(response.status_code)
    if response.status_code != 200:
        raise Exception(response.status_code, response.text)
    return response.text


def main():
    response = connect_to_endpoint(download_url)
    entries = response.splitlines()
    for entry in entries:
        print(entry)


if __name__ == "__main__":
    main()
